packages/components/src/duotone-picker/stories/duotone-picker.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { DuotonePicker } from '../';
+
+export default {
+	title: 'Components/DuotonePicker',
+	component: DuotonePicker,
+	argTypes: {
+		clearable: { control: { type: 'boolean' } },
+		onChange: { action: 'onChange' },
+		unsetable: { control: { type: 'boolean' } },
```

Comment:
that's a curious propname, wonder why this was preferred over more standard prop names like `readonly` or `disabled` 🤔

Reply:
Apparently it's the prop that dictates whether to add this Unset option in the options:

<img width="150" alt="The Unset option in the DuotonePicker" src="https://user-images.githubusercontent.com/555336/184832783-cc4b254e-e154-4768-bc2c-8cd82285553f.png">

Reply:
Ah, I missed it 😅